storybook/src/stories/notice.stories.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Notice } from '../src';
+import Notice from '../../../src/notice';
```

Comment:
I had to change to direct import because it wasn't working through the index.js file with named imports.